CHANGELOG.md

```diff
@@ -70,6 +70,12 @@
     - Rainbows
     - Sunspot
 
+
+- **Changes how the API method `set_transaction_name` works**
+
+  When the method `NewRelic::Agent.set_transaction_name` is called, it will now always change the name and category of the currently running transaction to what is passed in to the method. This is a change from how it functioned in previous agent versions. Previously, if set_transaction_name was called with a new transaction name and a new category that did not match the category that was already assignd to a transaction, neither the new name nor category would be saved to the transaction. 
```

Comment:
```suggestion
  When the method `NewRelic::Agent.set_transaction_name` is called, it will now always change the name and category of the currently running transaction to what is passed in to the method. This is a change from how it functioned in previous agent versions. Previously, if `set_transaction_name` was called with a new transaction name and a new category that did not match the category that was already assigned to a transaction, neither the new name nor category would be saved to the transaction. 
```

Reply:
oh awesome, thanks for catching that!